packages/beeq/src/global/styles/mixins/_keyframes.scss

```diff
@@ -24,6 +24,16 @@
   }
 }
 
+@keyframes move-indeterminate {
+  0% {
+    left: 0;
+  }
+
+  100% {
+    left: calc(100% - 80px);
+  }
+}
```

Comment:
We could move to the `progress.scss` file instead, since it's specific to that component, you can use the:
```css
@layer component {
  @keyframes move-indeterminate {
    0% {
      left: 0;
    }
  
    100% {
      left: calc(100% - 80px);
    }
  }
}
```

and then set the animation without the mixing (you can remove that):

```css
.progress-bar.progress-bar__border-shape::-moz-progress-bar {
  @apply rounded-full;
  
  animation: move-indeterminate 1s linear 0s infinite alternate-reverse;
}
```